packages/places/src/useCases/dbManagement/dbDeleter.js

```diff
@@ -0,0 +1,23 @@
+const Logger = require('../../utils/logger');
+
+const dbDeleter = (Model) => async (id) => {
+    try {
+        const data = await Model.update({
+            is_active: false
+        }, {
+            where: {
+                id
```

Comment:
What do you think to add `is_active: true` to this query? Because you are looking in to all the items, it does not matter if is already deleted or not.
```suggestion
                id,
                is_active: true,
```

Reply:
Thanks a lot, Jose, i did not think about it enough.
Done!